qaframework-bdd-tests/src/main/java/org/openmrs/contrib/qaframework/page/AddEditNewPrivilegePage.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ * 
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.contrib.qaframework.page;
+
+import org.openmrs.contrib.qaframework.helper.Page;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class AddEditNewPrivilegePage extends Page{
+	protected static final String PAGE_URL = "adminui/metadata/configureMetadata/privileges/managePrivileges.page/privilege.page?action=add&";
+	public static final By ENTER_PRIVILEGE_NAME = By.cssSelector("#privilege-field");
+	public static final By ENTER_PRIVILEGE_DESCRIPTION = By.cssSelector("#description-field");
+	public static final By SAVE_BUTTON = By.cssSelector("#save-button");
+	public static final By CANCEL_BUTTON = By.cssSelector("#privilegeForm input.cancel");
+	
+	public AddEditNewPrivilegePage(Page parent) {
+		super(parent);
+	}
+	
+	public AddEditNewPrivilegePage(Page parent, WebElement waitForStaleness) {
+		super(parent, waitForStaleness);
+	}
+	
+	public ManagePrivilegesPage enterPrivilegeName(String privilegeName){
+		findElement(ENTER_PRIVILEGE_NAME).clear();
+		findElement(ENTER_PRIVILEGE_NAME).sendKeys(privilegeName);
+		return new ManagePrivilegesPage(this);
+	}
+	
+	public ManagePrivilegesPage enterPrivilegeDescription(String privilegeDescription){
+		findElement(ENTER_PRIVILEGE_DESCRIPTION).clear();
+		findElement(ENTER_PRIVILEGE_DESCRIPTION).sendKeys(privilegeDescription);
+		return new ManagePrivilegesPage(this);
+	}
+	
+	public ManagePrivilegesPage clickSaveButton() {
+		clickOn(SAVE_BUTTON);
+		return new ManagePrivilegesPage(this);
+	}
+	
+	public void clickCancelButton() {
```

Comment:
Are you using this method any where in your step definition class.

Reply:
looks like an not